Documentation/PackageManagerCommunityProposal.md

```diff
@@ -306,6 +306,6 @@ In the future, the Swift Package Manager may allow packages to be built and inst
 
 If you have any questions about this document, or would like to share any thoughts about existing features, please contact the mailing list for Swift Package Manager development: <swift build-dev@swift.org>
 
-For information about contributing to Swift or the Swift Package Manager, check out ["Contributing to Swift" on Swift.org (https://swift.org/contributing).
+For information about contributing to Swift or the Swift Package Manager, check out "Contributing to Swift" on Swift.org (https://swift.org/contributing).
```

Comment:
Thanks! I think it was supposed to be a link, can you make it ["Contributing to Swift" on swift.org] (https://swift.org/contributing) instead?